### PR TITLE
Move to elastic-package check command

### PR DESCRIPTION
**Pull request by ruflin**
The check command can be used to have lint, format and build all in one. Updated the readme to use this command.

### CODE REVIEW

**Comment by ruflin** [comment]
@mrodm Here is the follow up PR for https://github.com/ruflin/ruflin-integration-package/pull/1

File: README.md [hunk 1]
```diff
@@ -216,7 +224,7 @@ Since our initial `0.0.1` version of the package, many modifications have been m
       link: http://fake-link
 ```
 
-Run `elastic-build` again and the upload command. The `0.1.0` version of the package will be updated to version 0.2.0. Only 1 version of the same package can be installed at the same time. But like this different versions of a package can be rolled out over time.
+Run `elastic-package check` again and the upload command. The `0.1.0` version of the package will be updated to version 0.2.0. Only 1 version of the same package can be installed at the same time. But like this different versions of a package can be rolled out over time.
```
Comment (mrodm) [inline README.md]: I was thinking to rephrase the first sentence, what about one of these ?
- Run `elastic-package check` again and upload the package.
- Run `elastic-package check` again and the install command.

**Review by mrodm — COMMENTED** [review]
Great documentation!

File: README.md (hunk 1 above)
Comment (ruflin) [reply README.md]: I went for: "Run `elastic-package check` again and the `elastic-package install` command"

**Comment by ruflin** [comment]
@mrodm Appreciate the review. I got this in quickly to get this moving but happy to make more updates to the doc. The shorter it gets, the better :-)

**Pull request merged**